spec/features/finding/index_details_spec.rb

```diff
@@ -2,24 +2,33 @@
 
 RSpec.feature "Index details" do
   scenario do
-    given_there_is_an_edition
+    given_there_is_an_edition_in_my_organisation
     when_i_visit_the_index_page
-    then_i_can_see_the_title
+    then_i_am_redirected_to_my_organisation_filter
```

Comment:
I'm never really sure how much we care about URLs in testing. But it seems like we're missing the behavioural aspect of this: where we don't see an edition that's not in our org.